PendingReleaseNotes.md

```diff
@@ -25,6 +25,9 @@
   - rgw: [gateway settings](Documentation/ceph-object-store-crd.md#gateway-settings)
   - nfs: [samples](Documentation/ceph-nfs-crd.md#samples)
 - When the operator is upgraded, the mgr and osd (not running on PVC) won't be restarted if the Rook binary version changes
+- DiscoverDaemonFilter made configurable for the user also by keeping the defaults intact
```

Comment:
I don't think `DiscoverDaemonFilter` is known, so this should be more user-friendly.